scenes/Ships/Ship_controls.gd

```diff
@@ -84,9 +85,17 @@ func handle_landing_request():
 	PlayerState.selection = closest_planet
 	PlayerState.selection.get_node("Selection").visible = true
 	
-	print (speed_check)
+	print ("Speed = ",speed_check)
 	if speed_check > 250:
 		print ("Moving too fast, slow down")
+	else:
+		var planetsize = closest_planet.get_node("Selection").get_size()
+		var distance_check = closest_planet.global_position.distance_to(global_position) - planetsize
+		print ("distance to planet ", distance_check)
+		if distance_check > 500:
```

Comment:
I think we should require that they get closer. A value of 150 here I think would be good.